internal/storage/storage_handle_test.go

```diff
@@ -23,202 +23,200 @@ import (
 
 	mountpkg "github.com/googlecloudplatform/gcsfuse/v2/internal/mount"
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/storage/storageutil"
-	"github.com/jacobsa/oglematchers"
 	. "github.com/jacobsa/ogletest"
+	"github.com/stretchr/testify/suite"
 )
 
 const invalidBucketName string = "will-not-be-present-in-fake-server"
 const projectID string = "valid-project-id"
 
-func TestStorageHandle(t *testing.T) { RunTests(t) }
-
 type StorageHandleTest struct {
+	suite.Suite
 	fakeStorage FakeStorage
 }
 
-var _ SetUpInterface = &StorageHandleTest{}
-var _ TearDownInterface = &StorageHandleTest{}
-
-func init() { RegisterTestSuite(&StorageHandleTest{}) }
+func TestStorageHandleSuite(t *testing.T) {
+	suite.Run(t, new(StorageHandleTest))
+}
 
-func (t *StorageHandleTest) SetUp(_ *TestInfo) {
+func (suite *StorageHandleTest) SetupSuite() {
 	var err error
-	t.fakeStorage = NewFakeStorage()
-	AssertEq(nil, err)
+	suite.fakeStorage = NewFakeStorage()
+	suite.Nil(err)
 }
 
-func (t *StorageHandleTest) TearDown() {
-	t.fakeStorage.ShutDown()
+func (suite *StorageHandleTest) TearDownSuite() {
+	suite.fakeStorage.ShutDown()
 }
 
-func (t *StorageHandleTest) invokeAndVerifyStorageHandle(sc storageutil.StorageClientConfig) {
+func (suite *StorageHandleTest) invokeAndVerifyStorageHandle(sc storageutil.StorageClientConfig) {
 	handleCreated, err := NewStorageHandle(context.Background(), sc)
-	AssertEq(nil, err)
-	AssertNe(nil, handleCreated)
+	suite.Nil(err)
+	suite.NotNil(handleCreated)
 }
 
-func (t *StorageHandleTest) TestBucketHandleWhenBucketExistsWithEmptyBillingProject() {
-	storageHandle := t.fakeStorage.CreateStorageHandle()
+func (suite *StorageHandleTest) TestBucketHandleWhenBucketExistsWithEmptyBillingProject() {
+	storageHandle := suite.fakeStorage.CreateStorageHandle()
 	bucketHandle := storageHandle.BucketHandle(TestBucketName, "")
 
-	AssertNe(nil, bucketHandle)
-	AssertEq(TestBucketName, bucketHandle.bucketName)
+	suite.NotNil(bucketHandle)
```

Comment:
any reason we are using suite. instead of assert. in all test methods ? one of the usp of this library is rich assert package, see [ref](https://github.com/stretchr/testify?tab=readme-ov-file#assert-package) here, any reason why we are not leveraging it ? asserts are commonly used in unit tests in all languages if both suite and assert are having same functionality, should prefer asserts.

Reply:
It is require to use suits as we have structure in the tests to create fake storage object, which will get created in setup before each tests. I found suits package is providing functionality same we are expecting in our current package. [ref](https://github.com/stretchr/testify?tab=readme-ov-file#suite-package)